README.rst

```diff
@@ -30,6 +40,60 @@ Xarray Extended with Climate Data Analysis Tools
 .. |Checked with mypy| image:: http://www.mypy-lang.org/static/mypy_badge.svg
    :target: http://mypy-lang.org/
 
+
+XCDAT is a Python library built on `xarray`_ for climate data analysis on structured grids.
+It serves as a spiritual successor to the Community Data Analysis Tools (`CDAT`_) library with a focus on long-term maintainability and extensibility.
+
+.. _xarray: https://github.com/pydata/xarray
+.. _CDAT: https://github.com/CDAT/cdat
+
+Planned Features
+-----------------
+
+XCDAT will provide climate domain specific features and utilities that are helpful for general xarray usage.
+
+These features include:
+
+- Support for metadata that is CF compliant, E3SM non-CF compliant, and common
+- Robust handling of coordinates and its associated bounds
+
+  - Name-agnostic retrieval of CF compliant coordinates and bounds using ``cf_xarray``
+  - Generating specific bounds or filling all missing bounds for supported axes
```

Comment:
I wonder if this should simply say "Generating bounds for supported axes (if they do not exist)"

When we say "filling all missing bounds" it makes me think that some grid cells have bounds and others do not (but I don't think that is the case).

Reply:
Good point. I called the method[ `fill_missing_bounds()`](https://xcdat.readthedocs.io/en/latest/_modules/xcdat/bounds.html#BoundsAccessor.fill_missing_bounds). Maybe it should be renamed to `add_missing_bounds()`.

Reply:
I like the `add_missing_bounds()`.